### ISSUE TITLE
Cannot subscribe to SSL/TLS Signal K stream with self signed certificate

### ISSUE THREAD

**Issue by cmotelet** [SignalK/signalk-server#965]
When subscribing to a Signal K SSL/TLS stream with a self-signed certificate on server the connection is refused with the message:
```
Feb 16 11:14:18 signalk-js-sdk/Connection [_onWSError] WS error self signed certificate +23ms
Feb 16 11:14:18 signalk-js-sdk/Connection [_onWSClose] called with wsURI: wss://192.168.200.12:3443/signalk/v1/stream?subscribe=none +3ms
```

**Comment by cmotelet** [comment]
Can be fixed with `export NODE_TLS_REJECT_UNAUTHORIZED=0` before starting Signal K server.
```
signalk@navmot3:~ $ cat .signalk/signalk-server
#!/bin/sh
export NODE_TLS_REJECT_UNAUTHORIZED=0
/usr/lib/node_modules/signalk-server/bin/signalk-server -c /home/signalk/.signalk $*
```

**Comment by cmotelet** [comment]
closed via #1122
